ngraph/core/src/op/experimental_detectron_roi_feature.cpp

```diff
@@ -0,0 +1,136 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
```

Comment:
change to 2017-2021. Please, verify other files.

Reply:
Fixed.